symengine/derivative.cpp

```diff
@@ -403,6 +402,12 @@ static RCP<const Basic> diff(const CLASS &self, \
         return mul(mul(self.rcp_from_this(), polygamma(zero, gamma_arg)), gamma_arg->diff(x));
     }
 
+    static RCP<const Basic> diff(const PolyGamma &self,
+            const RCP<const Symbol> &x) {
+        auto args = self.get_args();
+        return polygamma(add(args[0], one), args[1]);
```

Comment:
I think it should be like [this](https://github.com/nishnik/symengine/blob/ba6ccd834301622555a6e2936c6a42994cfaa02c/symengine/derivative.cpp#L435).